src/renderer/screens/exchange/Swap2/Form/FormSummary/index.js

```diff
@@ -9,10 +9,19 @@ import styled from "styled-components";
 import type { SwapTransactionType } from "~/renderer/screens/exchange/Swap2/utils/shared/useSwapTransaction";
 import type { SectionProviderProps } from "./SectionProvider";
 
-const Form: ThemedComponent<{}> = styled.section`
+const Form: ThemedComponent<{}> = styled.section.attrs(({ ready }) => ({
+  style: ready ? { opacity: 1, maxHeight: "100vh", overflow: "auto" } : {},
```

Comment:
`maxHeight: "100vh"` let's hope that the form height stays under the min page height 🙏